frontend/js/components/atoms/Hr.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Hr extends Component {
+    render() {
+        return (
+            <div className="hr">
+                <div className="hr-line"></div>
+                {
+                    this.props.label &&
+                    <label className="hr-label">{this.props.label}</label>
+                }
+            </div>
+        );
+    }
+}
+
+Hr.propTypes = {
+    label: PropTypes.string,
+    color: PropTypes.string
```

Comment:
The goal was that the developer could change the color of the HR where needed.  I'll update the PR.